front/package-lock.json

```diff
@@ -224,6 +224,97 @@
         "fsevents": "^2"
       }
     },
+    "../../cl2-component-library": {
```

Comment:
I wonder if we should revert this, it seems like it was left over from linking the package locally

Reply:
Hm... I'll reinstall again to see if it fixes it.

Reply:
Yep, looks good now 👍